apollo-federation/src/supergraph/mod.rs

```diff
@@ -713,16 +752,6 @@ fn extract_object_type_content(
             message: "@join__implements should exist for a fed2 supergraph".to_owned(),
         })?;
 
-    let context = supergraph_schema
-        .metadata()
-        .and_then(|metadata| metadata.for_identity(&Identity::context_identity()))
-        .and_then(|context_link| CONTEXT_VERSIONS.find(&context_link.url.version))
-        .and_then(|context_spec_def| {
```

Comment:
Bugs:
- When the context version is unknown, we should error out early (this is done now in `validate_supergraph_for_query_planning()`).